apps/web/src/pages/Skills/components/UpdateSkillShowcase.tsx

```diff
@@ -0,0 +1,422 @@
+                                      maxSkills: pageInfo.maxSkillCount,
+                                    },
+                                  ),
+                                  disabled: true,
+                                }
+                          }
+                          context="showcase"
+                          skills={skills}
+                          onSave={handleSave}
+                          showCategory={false}
+                          noToast
+                          {...(!canAdd
+                            ? {
+                                "data-h2-background": "base(background)",
+                                "data-h2-border-style": "base(dashed)",
+                                "data-h2-border-color": "base(gray.dark)",
+                                "data-h2-color": "base(gray.dark)",
+                                "data-h2-display": "base(flex)",
+                                "data-h2-justify-content": "base(center)",
+                                "data-h2-width": "base(100%)",
+                              }
+                            : {
+                                "data-h2-background":
+                                  "base(background) base:hover(secondary.10) base:focus-visible(focus)",
+                                "data-h2-border-style":
+                                  "base(dashed) base:focus-visible(solid)",
+                                "data-h2-border-color":
+                                  "base(secondary.darker) base:focus-visible(focus)",
+                                "data-h2-color":
+                                  "base(secondary.darker) base:focus-visible(black)",
+                                "data-h2-display": "base(flex)",
+                                "data-h2-justify-content": "base(center)",
+                                "data-h2-width": "base(100%)",
+                              })}
+                        />
+                      }
+                    >
+                      {fields.length ? (
+                        fields.map((item, index) => (
+                          <Repeater.Fieldset
+                            key={item.id}
+                            index={index}
+                            total={fields.length}
+                            onMove={move}
+                            onRemove={remove}
+                            legend={
+                              <span
+                                data-h2-display="base(flex)"
+                                data-h2-justify-content="base(space-between)"
+                              >
+                                <span>
+                                  {getLocalizedName(
+                                    getSkill(item.skill)?.name,
+                                    intl,
+                                  )}
+                                </span>
+                                <span>
```

Comment:
Looks like this should have a font-weight of 400 and a colour of `black.light`

Reply:
Thanks! Fixed by [b66f07f](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7933/commits/b66f07fc8078d74e92fd972ac6626f6ff8cb9100)